package/MDAnalysis/analysis/align.py

```diff
@@ -133,7 +137,7 @@
    >>> ref = mda.Universe(PSF, PDB_small)   # reference structure 1AKE
    >>> trj = mda.Universe(PSF, DCD)         # trajectory of change 1AKE->4AKE
    >>> alignment = align.AlignTraj(trj, ref, filename='rmsfit.dcd')
-   >>> alignment.run()
+   >>> alignment.run() # doctest:+SKIP
```

Comment:
This line produces output with a variable element. Use ellipsis as in https://stackoverflow.com/a/1024426 to also test for it.